packages/syft/src/syft/client/client.py

```diff
@@ -333,6 +333,13 @@ def get_node_metadata(self, credentials: SyftSigningKey) -> NodeMetadataJSON:
         else:
             return self.node.metadata.to(NodeMetadataJSON)
 
+    def to_blob_route(self, path: str, host=None) -> GridURL:
+        # TODO: FIX!
+        if host is not None:
+            return GridURL(host_or_ip=host, port=8333).with_path(path)
+        else:
+            return GridURL(port=8333).with_path(path)
```

Comment:
@shubham3121, I know we talked about this. I feel like I understand it well enough to patch the problem for situation but I dont know if I might be causing problems elsewhere, could we again have a look at this?